modules/gdscript/gdscript_editor.cpp

```diff
@@ -56,15 +56,15 @@ Ref<Script> GDScriptLanguage::get_template(const String &p_class_name, const Str
 
 	String _template = String() +
 					   "extends %BASE%\n\n" +
-					   "# class member variables go here, for example:\n" +
+					   "# Class member variables go here, for example:\n" +
```

Comment:
Bad comma. Some suggestions:

`Class member variables go here. For example:`
or
`Declare member variables here. Examples:`